pkg/controller/tailoredprofile/tailoredprofile_controller.go

```diff
@@ -188,44 +216,105 @@ func (r *ReconcileTailoredProfile) Reconcile(ctx context.Context, request reconc
 			}
 			return reconcile.Result{}, nil
 		}
-		var pbgetErr error
-		pb, pbgetErr = r.getProfileBundleFromRulesOrVars(instance)
-		if pbgetErr != nil && !common.IsRetriable(pbgetErr) {
-			// the Profile or ProfileBundle objects didn't exist. Surface the error.
-			err = r.handleTailoredProfileStatusError(instance, pbgetErr)
-			return reconcile.Result{}, err
-		} else if pbgetErr != nil {
-			return reconcile.Result{}, pbgetErr
+
+		// we will not use ProfileBundle for CustomRules
+		if len(customRules) <= 0 {
```

Comment:
Super minor nit we can fix in a follow up - len(customRules) == 0 should be sufficient I think?